back/transcendence_django/shared_models/constants.py

```diff
@@ -0,0 +1,5 @@
+from transcendence_django.dict_keys import LOCAL, LOSS, REMOTE, TIE, TOTAL, WIN
```

Comment:
I don't really get the point of having a shared_models directory instead of just placing the files within the already shared directory back/transcendence_django/transcendence_django.
But this file in particular doesn't really fit with the directory name. Put models.py and constants.py that are shared within the aforementioned directory instead?